src/pages/projects/FilterCheckboxItem.tsx

```diff
@@ -9,24 +9,26 @@ export default function FilterCheckboxItem({
   disabled,
 }: {
   label: string
-  checked: boolean
+  checked: boolean | null
```

Comment:
i've added this to allow a new "deselect all" option in the tags filter dropdown, that matches the style of other checkbox items but without itself including a checkbox